server/bin/pbench-dispatch.sh

```diff
@@ -257,7 +257,7 @@ while read tbmd5; do
         pbench-results-push ${tb} ${push_options}
         sts=${?}
         if [[ ${sts} -ne 0 ]]; then
-            log_error "${TS}: 'pbench-results-push ${tb} ${push_options}' failed, code ${sts}"
+            log_warn "${TS}: 'pbench-results-push ${tb} ${push_options}' failed, code ${sts}"
```

Comment:
Shouldn't this be an error?

Reply:
I assumed the point was that in the current configuration this failure doesn't affect the operation of the 0.69 server, even if it's not "correct". Once we drop the 0.69 archive and indexing, it'll be essential and a failure would become an error ...

Reply:
No, as @dbutenhof said, we are only "opportunistically" sending tar balls via the PUT API, that is, when it is configured.

In PR #3293, this is an error (see https://github.com/distributed-system-analysis/pbench/pull/3293/files#diff-8d37d0bc15d19991274a45df5bad8909f039d815e3bed7414e29d860dbba81dcR247).